front/app/utils/streams.ts

```diff
@@ -389,29 +394,23 @@ class Streams {
   // refetch the list from the back-end. Hence never manually push data in a query stream,
   // but always do a refetch!
   addStreamIdByDataIdIndex(
+    dataId: string,
     streamId: string,
```

Comment:
Swapping parameters of the same type is dangerous stuff, but I've adjusted function calls in the 2nd PR of streams.ts (which I'll merge into this one before merging with master).